packages/project-access/src/file/file-info.ts

```diff
@@ -0,0 +1,79 @@
+import { JSONFileInfo } from "../types";
+
+type WriteJsonSpace = number | string;
+const CHAR_SPACE = ' ';
+const CHAR_TAB = '\t';
+
+/**
+ * Method calculates tab spacing parameter for 'JSON.stringify' method.
+ *
+ * @param tabInfo - External tab configuration.
+ * @returns tab size information.
+ */
+export function getJsonSpace(tabInfo: JSONFileInfo): WriteJsonSpace | undefined {
+    let space: WriteJsonSpace | undefined;
+    // 'tabInfo' exists - it was passed as custom configuration or calculated from target file
+    if (tabInfo.useTabSymbol) {
+        // Tab symbol should be used as tab
+        space = CHAR_TAB.repeat(tabInfo.size ?? 1);
+    } else {
+        // Spaces should be used as tab
+        space = tabInfo.size;
+    }
+    return space;
+}
```

Comment:
If we simplify the structure to have `space` property, this function would be obsolete.